### PR TITLE
runtime-tools: add ROADMAP.md

### PR DESCRIPTION
**Pull request by Mashimiao**
We should have a roadmap which helps us clear goals.

Signed-off-by: Ma Shimiao <mashimiao.fnst@cn.fujitsu.com>

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# OCI runtime-tools Roadmap
+
+This document serves to provide a long term roadmap on our quest to a 1.0 version of the OCI runtime-tools.
+Its goal is to help both maintainers and contributors find meaningful tasks to focus on and create a low noise environment.
```
Comment (liangchenye) [inline ROADMAP.md]: to help maintainers and contributors to find

File: ROADMAP.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+Bundle structure and all fileds in config.json should can be validated by oci-runtime-validate.
+
+*Owner:*
+
+### Runtime Validation
+
+Validate a runtime whether meets requirments of OCI runtime specs.
+
+runtime-tools should can validate a runtime from aspects of contianer lifecycle, operations  and application of [container configuration files][runtime-spec-config] (like cgroups setting, devices setting, etc).
```
Comment (liangchenye) [inline ROADMAP.md]: runtime-tools should validate a runtime from aspects of container lifecycle, operations

File: ROADMAP.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+All fields defined in [Container Configuration file][runtime-sepc-config] should be support by oci-runtime-generate.
+
+*Owner:*
+
+### Bundle Validation
+
+Validate an OCI runtime bundle.
+
+Bundle structure and all fileds in config.json should can be validated by oci-runtime-validate.
```
Comment (liangchenye) [inline ROADMAP.md]: should be

**Comment by wking** [comment]
On Wed, Mar 08, 2017 at 01:03:16AM -0800, Ma Shimiao wrote:
> We should have a roadmap which helps us clear goals.

I'd rather track this sort of thing with GitHub issues, maybe with a
“roadmap” label, and milestones.  The spec has a ROADMAP.md, and as
far as I can tell it's just abandoned cruft [1].  It also links an
abandoned wiki page [2].

[1]: https://github.com/opencontainers/runtime-spec/commits/dd41d8ffd6dd29fc0be1f3197519f11727cb19d3/ROADMAP.md
[2]: https://github.com/opencontainers/runtime-spec/wiki/RoadMap:

File: ROADMAP.md (hunk 1 above)
Comment (Mashimiao) [inline ROADMAP.md]: fixed

**Comment by Mashimiao** [comment]
We should have milestones, but milestones and roadmap I think they play different roles.
roadmap clearly tells us what we need to do, milestone looks like a schedule.
If we maintain ROADMAP.md well, it will make great sense.

**Pull request closed without merging**